android/capacitor-plugin-file-transfer/build.gradle

```diff
@@ -42,10 +42,10 @@ repositories {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'ionic-team:capacitor-android:1+'
-    compile 'com.squareup.okhttp3:okhttp:3.11.0'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.1'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    implementation 'ionic-team:capacitor-android:1.2.1'
+    implementation 'com.squareup.okhttp3:okhttp:3.14.0'
+    implementation 'junit:junit:4.12'
```

Comment:
Is there any particular reason why this had to change from `testImplementation` to `implementation`?

Reply:
That was a mistake. I don't remember doing it!